README.md

```diff
@@ -62,7 +62,19 @@ Visit [RBAC permissions for running the tasks](docs/tasks-rbac-permissions.md) i
 
 ## Examples
 
-#### [Unit Tester Pipeline](examples/pipelines/unit-tester) 
+#### [Windows bios Installer Pipeline](pipelines/windows-bios-installer)
+
+Downloads a Windows ISO file into a PVC and automatically installs Windows by using a custom Answer file into a new base DV.
+
+#### [Windows efi Installer Pipeline](pipelines/windows-efi-installer)
+
+Downloads a Windows ISO file into a PVC and automatically installs Windows 11, or 2k22 with EFI enabled by using a custom Answer file into a new base DV.
+
+#### [Windows Customize Pipeline](pipelines/windows-customize)
+
+Applies customizations to an existing Windows 10, 11, 2k22 installation by using a custom Answer file and creates a new base DV.
```

Comment:
Not limited to specific Windows versions, is it?

Reply:
This pipeline is only tested against win10, 11 and 2k22. Other versions are not tested

Reply:
You could generalize it and add a paragraph which lists the tested versions?

Reply:
added